renovate.json

```diff
@@ -27,15 +27,17 @@
     },
     {
       "groupName": "k8s.io",
-      "matchPackagePrefixes": [
-        "k8s.io",
-        "sigs.k8s.io"
-      ],
-      "schedule": [
-        "every weekend"
-      ],
-      "allowedVersions": "< 1.0.0"
-    }
+      "matchPackagePatterns": ["^k8s.io"],
+￼     "excludePackagePatterns": ["^k8s.io/kube-openapi"],
```

Comment:
it seems there is whitespace damage

Reply:
yes there was some strange char at the line start